tasks.py

```diff
@@ -118,78 +186,155 @@ def cli(context):
 
-@task
-def flake8(context):
+    if is_truthy(local):
+        context.run(command)
+    else:
+        docker_compose(
+            context,
+            f"run --entrypoint '{command}' nautobot",
+            pty=True,
+        )
+
+
+@task(help={"local": "run this task locally vs inside the docker container (default: False)"})
+def flake8(context, local=None):
     """Check for PEP8 compliance and other style issues."""
-    docker_compose(context, "run --entrypoint 'flake8 development/ nautobot/ tasks.py' nautobot", pty=True)
+    command = "flake8 development/ nautobot/ tasks.py"
+
+    if local is None:
+        local = context.nautobot.local
+    if is_truthy(local):
+        context.run(command)
+    else:
+        docker_compose(context, f"run --entrypoint '{command}' nautobot", pty=True)
+
+
+@task(help={"local": "run this task locally vs inside the docker container (default: False)"})
+def check_migrations(context, local=None):
+    """Check for missing migrations."""
+    command = "nautobot-server --config=nautobot/core/tests/nautobot_config.py makemigrations --dry-run --check"
+
+    if local is None:
+        local = context.nautobot.local
+    if is_truthy(local):
+        context.run(command)
+    else:
+        docker_compose(context, f"run --entrypoint '{command}' nautobot", pty=True)
 
 
 @task(
     help={
         "keepdb": "save and re-use test database between test runs for faster re-testing.",
         "label": "specify a directory or module to test instead of running all Nautobot tests",
+        "failfast": "fail as soon as a single test fails don't run the entire test suite",
+        "local": "run this task locally vs inside the docker container (default: False)",
     }
 )
-def unittest(context, keepdb=False, label="nautobot"):
+def unittest(context, keepdb=False, label="nautobot", failfast=False, local=None):
     """Run Nautobot unit tests."""
-    command = f"run --entrypoint 'coverage run -m nautobot.core.cli test {label} --config=nautobot/core/tests/nautobot_config.py"
+    command = f"coverage run -m nautobot.core.cli test {label} --config=nautobot/core/tests/nautobot_config.py"
+
+    if local is None:
+        local = context.nautobot.local
     if keepdb:
         command += " --keepdb"
-    command += "' nautobot"
-    docker_compose(context, command, pty=True)
+    if failfast:
+        command += " --failfast"
+    if is_truthy(local):
+        context.run(command)
+    else:
+        docker_compose(context, f"run --entrypoint '{command}' nautobot", pty=True)
 
 
-@task
-def unittest_coverage(context):
+@task(help={"local": "run this task locally vs inside the docker container (default: False)"})
+def unittest_coverage(context, local=None):
     """Report on code test coverage as measured by 'invoke unittest'."""
-    docker_compose(context, "run --entrypoint 'coverage report --skip-covered --omit *migrations*' nautobot", pty=True)
+    command = "coverage report --skip-covered --omit *migrations*"
```

Comment:
This needs to be updated to run the same flags as was being done in `cibuild.sh`:

```
--skip-covered --include "nautobot/*" --omit "*migrations*"
```

Without `--include "nautobot/*", it's doing a coverage report on every single file in the environment. You can see it starting at line 1098 here: https://travis-ci.com/github/nautobot/nautobot/jobs/495740274#L1098.

Reply:
This wasn't resolved. *cough*

Reply:
```suggestion
    command = "coverage report --skip-covered --include "nautobot/*" --omit *migrations*"
```